server/server.go

```diff
@@ -153,6 +153,13 @@ type Config struct {
 
 	// This limiter is called for every started and finished gRPC request.
 	GrpcMethodLimiter GrpcInflightMethodLimiter `yaml:"-"`
+
+	ThroughputConfig ThroughputConfig `yaml:"throughput_config"`
```

Comment:
`throughput_config` is the name of the config. The `_config` suffix is a bit redundant.